package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "angular-testcafe",
+  "name": "angular-testcafe-sword",
```

Comment:
Please remove the `package-lock.json` file. This module will be used the latest version of packages.